frontend/simple/sass/_groupincome.scss

```diff
@@ -288,4 +288,172 @@ tr.paid td .date span.paid {
+  color: white;
+}
+.symbol{
+  vertical-align: bottom;
+  margin: 10px 0;
+}
+.question-group .notification .select:after {
+  border: 1px solid #69707a;
+  border-right: 0;
+  border-top: 0;
+}
+.center {
+  margin: auto auto
+}
+.percentage{
+  height: 26px;
+}
+.question-group .notification {
+  display: block;
+  padding: 30px;
+  margin: 10px 5px;
+  text-align: center;
+}
+.dotted{
+  vertical-align: bottom;
+  background-color: transparent;
+  border: none;
+  margin: 10px;
+  padding: 10px 10px 0 10px;
+  border-bottom: 0.2rem dashed #69707a;
+  font-size: 20px;
+  width: 70%;
+}
+.is-info textarea.dotted{
+  border: 0.2rem dashed white;
+  color: white;
+}
+
+.percentage{
+  background-color: transparent;
+  color: inherit;
+  border: none;
+  outline:none;
+  font-size: 28px;
+  border: 1px;
+}
+
+textarea.dotted{
+  border: 0.2rem dashed #69707a;
+  height: 150px;
+  width: 70%;
+}
+.notfication.is-info textarea.dotted{
+  color: white;
+  border-color: white;
+}
+input[type=range]{
+  background-color: blue
+}
+/* Slider toggle Styles */
+/* The switch - the box around the slider */
+.switch {
+  position: relative;
+  display: inline-block;
+  width: 60px;
+  height: 34px;
+}
+
+/* Hide default HTML checkbox */
+.switch input {display:none;}
+
+/* The slider */
+.slider {
+  position: absolute;
+  cursor: pointer;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  background-color: #ccc;
+  -webkit-transition: .4s;
+  transition: .4s;
+}
+
+.slider:before {
+  position: absolute;
+  content: "";
+  height: 26px;
+  width: 26px;
+  left: 4px;
+  bottom: 4px;
+  background-color: white;
+  -webkit-transition: .4s;
+  transition: .4s;
+}
+
+input:checked + .slider {
+  background-color: #2196F3;
+}
+
+input:focus + .slider {
+  box-shadow: 0 0 1px #2196F3;
+}
+
+input:checked + .slider:before {
+  -webkit-transform: translateX(26px);
+  -ms-transform: translateX(26px);
+  transform: translateX(26px);
+}
+
+/* Rounded sliders */
+.slider.round {
+  border-radius: 34px;
+}
+
+.slider.round:before {
+  border-radius: 50%;
+}
+/* end slider toggle styles */
+/* rounded range controls */
+input[type=range] {
+  -moz-appearance: none;
+  -webkit-appearance: none;
+  background-color: #ccc;
+  border-radius: 34px;
+  padding: 4px;
+  height: 34px;
+}
+
+input[type=range]:focus {
+  outline: none;
+}
+
+input[type=range]::-webkit-slider-thumb {
+  -webkit-appearance: none;
+  border-radius: 50%;
+  height: 26px;
+  width: 26px;
+  background: #ffffff;
+}
+
+input[type=range]::-moz-range-thumb {
+  -moz-appearance: none;
+  border-radius: 50%;
+  height: 26px;
+  width: 26px;
+  background: #ffffff;
+  border: none;
+}
+
+input[type=range]::-moz-range-track {
+  background: transparent;
+  border-color: transparent;
+  color: transparent;
+}
+/* end rounded range control*/
+.full-screen, html, body, #app{
+  height: 100%;
+}
+.created{
+  color: #97cd76;
+  display: block;
+  font-size: 1rem;
+  margin-top: 5px;
+  text-align: center;
```

Comment:
Woah, OK, a few notes:

1. This is way more new CSS being added than I expected. To be clear, we want to save your time and our $ not on generating new CSS but on using only the existing CSS/SASS (as much as possible) that's provided by Bulma. Remember, for the final UI, we're going to have a professional designer go over this stuff, so any effort on making custom sliders and stuff is just going to be wasted.
2. Do not write out browser-specific extension stuff yourself. Always use a SASS/SCSS things to auto-generate them for you. If Bulma doesn't have them, then look at Compass and copy some of the mixins/functions from that to do it (or write them yourself). This is part of DRY principle.